From 30ea52f88cddd99825b6bb9be79b738702cb1a6a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ga=C3=ABl=20PORTAY?= <gael.portay@collabora.com>
Date: Tue, 24 Sep 2019 23:00:35 -0400
Subject: [PATCH] dracut.modules.7.asc: fix reference to insmodpost module

The module 96insmodpost was renamed to 90kernel-modules since commit
5078c98a (move insmodpost and blacklisting to 90kernel-modules)
---
 dracut.modules.7.asc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/dracut.modules.7.asc b/dracut.modules.7.asc
index fb30409e..81135cff 100644
--- a/dracut.modules.7.asc
+++ b/dracut.modules.7.asc
@@ -146,8 +146,8 @@ FIXME
 
 == Writing a Module
 
-A simple example module is _96insmodpost_, which modprobes a kernel module after
-udev has settled and the basic device drivers have been loaded.
+A simple example module is _90kernel-modules_, which modprobes a kernel module
+after udev has settled and the basic device drivers have been loaded.
 
 All module installation information is in the file module-setup.sh.
 

